src/main/java/com/github/vkorobkov/jfixtures/fluent/impl/JFixturesResultImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.github.vkorobkov.jfixtures.fluent.impl;
+
+import com.github.vkorobkov.jfixtures.config.ConfigLoader;
+import com.github.vkorobkov.jfixtures.fluent.JFixturesResult;
+import com.github.vkorobkov.jfixtures.instructions.CleanTable;
+import com.github.vkorobkov.jfixtures.instructions.CustomSql;
+import com.github.vkorobkov.jfixtures.instructions.InsertRow;
+import com.github.vkorobkov.jfixtures.instructions.Instruction;
+import com.github.vkorobkov.jfixtures.loader.FixturesLoader;
+import com.github.vkorobkov.jfixtures.processor.Processor;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.val;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElements;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
+
+@NoArgsConstructor
+@XmlRootElement(name = "instructions")
+abstract class JFixturesResultImpl implements JFixturesResult {
+    private String fixturesFolder;
+
+    @Setter
```

Comment:
Why do we need `@Setter`? Can't we put XML related annotation on getter(`getInstructions()` method) ?

Reply:
If getter is an allowed mechanism, then, perhaos, we can remove this field at all or leave only `getInstructions()`?